.github/actions/host-init/action.yml

```diff
@@ -0,0 +1,45 @@
+name: "Host Initialization"
+description: "Sets up the host environment, including authentication and dependencies."
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Service Account
+      uses: google-github-actions/auth@v2
+      with:
+        project_id: 'e2b-prod'
+        # TODO: Move the identity provider construction to Terraform (like secrets.workload_identity_provider)
```

Comment:
Maybe change the todo, to get rid of dependance on gcp storage bucket and to be able to use local storage